apps/aecore/src/aec_block_micro_candidate.erl

```diff
@@ -210,7 +210,7 @@ add_txs_to_trees(MaxGas, Trees, Txs, Env) ->
 add_txs_to_trees(_MaxGas, Trees, [], Acc,_Env) ->
     {lists:reverse(Acc), Trees};
 add_txs_to_trees(MaxGas, Trees, [Tx | Txs], Acc, Env) ->
-    TxGas = aetx:gas(aetx_sign:tx(Tx)),
+    TxGas = aetx:gas(aetx_sign:tx(Tx), aetx_env:height(Env)),
     case TxGas =< MaxGas of
```

Comment:
I see that `TxGas` may be zero though in that case transaction will be grecefully skipped with reason `{error, too_low_abs_ttl}` by non-strict call `aec_trees:apply_txs_on_state_trees` below. Ok.